include/zephyr/usb/class/usbd_midi.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Titouan Christophe
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_USB_CLASS_USBD_MIDI_H_
+#define ZEPHYR_INCLUDE_USB_CLASS_USBD_MIDI_H_
+
+/**
+ * @brief USB-MIDI 2.0 class device API
+ * @defgroup usb_midi USB MIDI 2.0 Class device API
+ * @ingroup usb
+ * @since 4.1
+ * @version 0.1.0
+ * @see midi20: "Universal Serial Bus Device Class Definition for MIDI Devices"
+ *              Document Release 2.0 (May 5, 2020)
+ * @{
+ */
+
+#include <zephyr/device.h>
+
```

Comment:
Could you please add the 
```
#ifdef __cplusplus
extern "C" {
#endif

```
with corresponding
```
#ifdef __cplusplus
}
#endif
```
sections to the header to suppress name mangling in C++? Like in "usbd_hid.h"...